conf/arm.config

```diff
@@ -0,0 +1,273 @@
+    }
+
+    withName: 'UMITOOLS_DEDUP' {
+        container = { workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/ba/ba7c02244236db73a56ff1eb880b16c2dd53679834e1b72bb096e1f633e35644/data' : 'community.wave.seqera.io/library/umi_tools:1.1.5--b2721816f7a92564' }
+    }
+
+    withName: 'GFFREAD' {
+        container = { workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/9e/9e6db95013607b07689e38ee37a654d029236de77fdfde97fe1866f45d01e064/data' : 'community.wave.seqera.io/library/gffread:0.12.7--1577aa7c95340d9f' }
+    }
+
+    withName: 'FASTP' {
+        container = { workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/7e/7ec6234f24305b9694b89de0352fff942ce1d68f840184020b0b9135bc54bda1/data' : 'community.wave.seqera.io/library/fastp:0.23.4--c1051f83dcae53f5' }
+    }
+
+    withName: 'SALMON_QUANT' {
+        container = { workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/b8/b8e3bcb1daa42497d0159cd5ae3b6d164b4470a972dd72d9d10b365fa166e5a4/data' : 'community.wave.seqera.io/library/salmon:1.10.3--dc33937abc5bffd1' }
+    }
+
+    withName: 'SALMON_INDEX' {
+        container = { workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/b8/b8e3bcb1daa42497d0159cd5ae3b6d164b4470a972dd72d9d10b365fa166e5a4/data' : 'community.wave.seqera.io/library/salmon:1.10.3--dc33937abc5bffd1' }
+    }
+
+    withName: 'GUNZIP' {
+        container = { workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/4b/4bcb36d5e1fb90c8af0171a83b792ada446b04feb4ac3f26d0ce550d3c678383/data' : 'community.wave.seqera.io/library/grep_sed_tar:4b2523b1c76a5494' }
+    }
+
+    withName: 'TRIMGALORE' {
+        container = { workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/10/10e7b91ef06bdf5790120809afc8cf36aac0261b618d810c952d5afd60dd5ed2/data' : 'community.wave.seqera.io/library/cutadapt_trim-galore:2aa2fc797328573b' }
+    }
+
+    //
+    // Tools that do not have native Conda builds for AMD and ARM. Required custom builds in 'seqera' Conda channel.
+    //
+
+    withName: '.*:ALIGN_STAR:STAR_ALIGN_IGENOMES' {
+        conda = 'seqera::star=2.6.1d bioconda::samtools=1.21 conda-forge::gawk=5.1.0'
```

Comment:
Could not the `conda` definitions go in the corresponding module?

Reply:
Ideally not, we're trying to keep the non-standard bits out of the main body of the workflow code.

Hacks like this should stay here until they can be eliminated cleanly https://github.com/nf-core/rnaseq/issues/1431.

Reply:
didn't realised it was `seqera` channel